sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -263,6 +265,74 @@ public enum SasProtocol
+        public string Permissions { get { throw null; } }
+        public Azure.Data.Tables.Sas.SasProtocol Protocol { get { throw null; } set { } }
+        public Azure.Data.Tables.Sas.TableAccountSasResourceTypes ResourceTypes { get { throw null; } set { } }
+        public System.DateTimeOffset StartsOn { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public void SetPermissions(Azure.Data.Tables.Sas.TableAccountSasPermissions permissions) { }
+        public void SetPermissions(string rawPermissions) { }
+        public string Sign(Azure.Data.Tables.TableSharedKeyCredential sharedKeyCredential) { throw null; }
+        public Azure.Data.Tables.Sas.TableAccountSasQueryParameters ToSasQueryParameters(Azure.Data.Tables.TableSharedKeyCredential sharedKeyCredential) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override string ToString() { throw null; }
+    }
+    [System.FlagsAttribute]
+    public enum TableAccountSasPermissions
+    {
+        All = -1,
+        Read = 1,
+        Write = 2,
+        Delete = 4,
+        List = 8,
+        Add = 16,
+        Update = 64,
+    }
+    public sealed partial class TableAccountSasQueryParameters
+    {
+        internal TableAccountSasQueryParameters() { }
+        public const string DefaultSasVersion = "2019-07-07";
+        public static Azure.Data.Tables.Sas.TableAccountSasQueryParameters Empty { get { throw null; } }
+        public System.DateTimeOffset ExpiresOn { get { throw null; } }
+        public string Identifier { get { throw null; } }
+        public Azure.Data.Tables.Sas.SasIPRange IPRange { get { throw null; } }
+        public string Permissions { get { throw null; } }
+        public Azure.Data.Tables.Sas.SasProtocol Protocol { get { throw null; } }
+        public string Resource { get { throw null; } }
+        public Azure.Data.Tables.Sas.TableAccountSasResourceTypes? ResourceTypes { get { throw null; } }
+        public Azure.Data.Tables.Sas.TableAccountSasServices? Services { get { throw null; } }
+        public string Signature { get { throw null; } }
+        public System.DateTimeOffset StartsOn { get { throw null; } }
+        public string Version { get { throw null; } }
+        public override string ToString() { throw null; }
+    }
+    [System.FlagsAttribute]
+    public enum TableAccountSasResourceTypes
+    {
+        All = -1,
+        Service = 1,
+        Container = 2,
+        Object = 4,
+    }
+    [System.FlagsAttribute]
+    public enum TableAccountSasServices
```

Comment:
It depends...  if you're keeping `TableAccountSasQueryParameters` and adding a `TableUriBuilder`, then you'll need all the other values because someone may give you a SAS that works with Tables and Blobs that you'll need to be able to take apart.  If you're not going to add `TableUriBuilder`, then we can nix all the `SasQueryParameters` classes and add `*SasBuilder.Sign(...)` methods.

Reply:
Good point! I had forgotten that much of the `*Parameters` class is to re-parse an existing token string.